code/modules/mob/living/simple_animal/clockwork_constructs.dm

```diff
@@ -0,0 +1,123 @@
+#define MARAUDER_SHIELD_MAX 4
+
+/mob/living/simple_animal/hostile/clockwork_construct/clockwork_marauder
+	name = "clockwork marauder"
+	desc = "A brass machine of destruction,"
+	icon = 'icons/mob/clockwork_mobs.dmi'
+	icon_state = "clockwork_marauder"
+	icon_living = "clockwork_marauder"
+	icon_dead = "clockwork_marauder_dead"
+	a_intent = INTENT_HARM
+	stop_automated_movement = TRUE
+	see_in_dark = 8
+	health = 140
+	maxHealth = 140
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	initial_traits = list(TRAIT_FLYING)
+	move_resist = MOVE_FORCE_OVERPOWERING
+	mob_size = MOB_SIZE_LARGE
+	pass_flags = PASSTABLE
+	damage_coeff = list("brute" = 1, "fire" = 1, "tox" = 0, "clone" = 0, "stamina" = 0, "oxy" = 0)
+	obj_damage = 80
+	melee_damage_lower = 24
+	melee_damage_upper = 24
+	attacktext = "slices"
+	attack_sound = 'sound/weapons/bladeslice.ogg'
+	chat_color = "#CAA25B"
+	retreat_distance = 2 // Dodge in and out of combat.
+	faction = list("hostile") // We hate Nar'sie here, but no clockcult yet.
+	pressure_resistance = 100
+	universal_speak = TRUE
+	AIStatus = AI_OFF
+	loot = list(/obj/item/food/ectoplasm, /obj/item/clockwork/component/geis_capacitor/fallen_armor)
```

Comment:
ectoplasm?

Reply:
So, the nar'sie constructs drop the stuff, and I figured that the ectoplasm would be what happens when the dark flames inside the armour sputter and condense. I can remove it if that is preferable.